azure-pipelines.yml

```diff
@@ -1,17 +1,11 @@
 trigger:
 - master
+- pyfilesystem-s3fs-support
```

Comment:
change to match this block:
https://github.com/finos/perspective/blob/master/azure-pipelines.yml#L1

Reply:
sure

Reply:
done